extensions_admin/pulp_rpm/extensions/admin/rpm_repo/pulp_cli.py

```diff
@@ -39,6 +39,8 @@ def initialize(context):
     copy_section.add_command(copy_commands.PackageEnvironmentCopyCommand(context))
     copy_section.add_command(copy_commands.AllCopyCommand(context))
     copy_section.add_command(copy_commands.SrpmCopyCommand(context))
+    copy_section.add_command(copy_commands.YumRepoMetadataFileCommand(context))
+    copy_section.add_command(copy_commands.DrpmCopyCommand(context))
```

Comment:
It turns out this command already existed but just hadn't been added to the structure!